### DISCUSSION TITLE
Exposing an API for macros?

### DISCUSSION THREAD

**Discussion by Eryx5502** [AnimaBeyondDevelop/AnimaBeyondFoundry#183] [Ideas]
Some changes proposed in PR #181 brought attention to the need of exposing some api which could be used from Macros. In particular, this came from the need of @paiva92 to roll psychic potential from a macro. We realised that this particular problem could be solved creating a dedicated `ABFRoll` subclass, since rolls are available to macros.

But it might be interesting to discuss whether other things from the system might be needed for macros, and whether we should expose an API for that purpose. Anyone used to write macros has any idea/request? Does this even make sense?

**Comment by Linkaynn** [comment]
At this point, ABFRoll is enough for what people is used to do, what about close this conversation and open it when more people is interested on this?
